launch/launch/actions/unset_environment_variable.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the UnsetEnvironmentVariable action."""
+
+import os
+
+from typing import List
+
+from ..action import Action
+from ..launch_context import LaunchContext
+from ..some_substitutions_type import SomeSubstitutionsType
+from ..substitution import Substitution
+from ..utilities import normalize_to_list_of_substitutions
+from ..utilities import perform_substitutions
+
+
+class UnsetEnvironmentVariable(Action):
+    """ Action that unsets an environment variable """
```

Comment:
doc nitpick: please do not use extra leading/trailing spaces and use punctuation

Also, it is worth noting that this does not fail if the environment variable does not exist.

For example:

```
"""Action that unsets an environment variable if it is set otherwise, does nothing."""
```

Reply:
Changed.